test/unit/server.js

```diff
@@ -211,6 +211,61 @@ describe('Server', function () {
     });
   });
 
+  describe('_unsubscribe', function () {
+    var server;
+    var queues = ['a'];
+
+    beforeEach(function () {
+      // runnable-hermes 6.1.0 introduced .getQueues
+      sinon.stub(hermes, 'hermesSingletonFactory').returns({
+        unsubscribe: sinon.stub(),
+        getQueues: sinon.stub().returns(queues)
+      });
+      server = new ponos.Server({ queues: queues });
+      server.setAllTasks({ a: noop });
+    });
+
+    afterEach(function () {
+      hermes.hermesSingletonFactory.restore();
+    });
+
+    it('should apply the correct callback for the given queue', function () {
+      server._unsubscribe('a');
```

Comment:
this returns a promise, yes? you should assert on it :) otherwise, it could throw and cause a weird error in the tests